CONTRIBUTING.md

```diff
@@ -52,6 +52,13 @@ different changes to different files. For example, for someone making a weather
 ```
 git checkout -b "weather-plugin"
 ```
+test if all files are well formatted, complying with style and security rules, before send the PR
+
+```
```

Comment:
Add instructions to install pre-commit instead of running each tool separately

Reply:
I've added the instructions